scripts/apps/archive/directives/ItemPreviewContainer.ts

```diff
@@ -6,6 +11,22 @@ export function ItemPreviewContainer() {
             scope.item = null;
 
             scope.$on('intent:preview:item', (event, intent) => {
+                if (itemInPreviewMode != null) {
+                    window.dispatchEvent(
+                        new CustomEvent(getCustomEventNamePrefixed('articlePreviewEnd'), {detail: itemInPreviewMode}),
+                    );
```

Comment:
seems like quite some boilerplate, would it be possible to do something like:
```ts
import {dispatchEvent} from 'superdesk';

dispatchEvent('articlePreviewEnd', {...});
```
?